SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -15,14 +15,27 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 public:
 	USpatialGDKEditorSettings(const FObjectInitializer& ObjectInitializer);
 
+	virtual void PostEditChangeProperty(struct FPropertyChangedEvent& PropertyChangedEvent) override;
+	virtual void PostInitProperties() override;
+
 private:
 	/** Path to the directory containing the SpatialOS-related files. */
 	UPROPERTY(EditAnywhere, config, Category = "General", meta = (ConfigRestartRequired = false, DisplayName = "SpatialOS directory"))
 	FDirectoryPath SpatialOSDirectory;
 
+public:
+	/** If checked, all dynamically spawned entities will be deleted when server workers disconnect. */
+	UPROPERTY(EditAnywhere, config, Category = "Play in editor settings", meta = (ConfigRestartRequired = false, DisplayName = "Delete Dynamically Spawned Entities"))
+	bool bDeleteDynamicEntities;
+
+	/** Generate default launch config. */
```

Comment:
The comment is not useful as it's rehashing what the variable name is. Give more info please.